clients/src/main/java/org/apache/kafka/clients/FetchSessionHandler.java

```diff
@@ -67,6 +67,10 @@ public FetchSessionHandler(LogContext logContext, int node) {
         this.node = node;
     }
 
+    public int sessionId() {
```

Comment:
Nit: `()` is not needed for methods that don't have side effects.

Reply:
This is Java

Reply:
Yes, my bad.